README.md

```diff
@@ -29,7 +29,7 @@ The following is required to properly run the cloud server.
 1. Install [Go](https://golang.org/doc/install)
 2. Install [Terraform](https://learn.hashicorp.com/terraform/getting-started/install.html) version v0.11.14
```

Comment:
Looks like we need to update the version recommendation here as well.

Reply:
You are absolutely right! thanks for the suggestion!